package.json

```diff
@@ -38,47 +38,51 @@
     "generate-docs": "api-documenter markdown --input-folder temp --output-folder docs && rm -rf temp",
     "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite",
     "prepublishOnly": "npm run build",
-    "test": "jest --coverage",
-    "storybook": "start-storybook -p 8080",
-    "build-storybook": "build-storybook"
+    "storybook": "start-storybook -p 6006",
```

Comment:
is this port change fine or does it change something for Sites?

Reply:
I believe live preview expects port 8080. Updated the command in ci_config for live preview.

Reply:
I believe live preview expects port 8080. This port change was made directly to storybook-site branch previously (https://github.com/yext/search-ui-react/commit/8d5f6969b99b73916bcab814ba8a87cec1a5e78b). Updated the command in ci_config for live preview.